suppress duplicate reports with -c/-C and -q/-Q

auerswal opened the pull request:
If fping receives a duplicate Echo Reply packet, it reports this.  This report was suppressed for loop mode, and for other operating modes where each receive event is reported. Now, this duplicate report is also suppressed when in count mode with options -q, --quiet or -Q, --squiet.

This addresses GitHub issue #389.

auerswal committed:
suppress duplicate reports with -c/-C and -q/-Q

If fping receives a duplicate Echo Reply packet, it reports
this.  This report was suppressed for loop mode, and for
other operating modes where each receive event is reported.
Now, this duplicate report is also suppressed when in count
mode with options -q, --quiet or -Q, --squiet.

This addresses GitHub issue #389.

auerswal commented:
I have created this as a draft, because I am not sure if there was a reason to treat loop mode and count mode differently regarding duplicate reporting.

auerswal commented:
Any reasons against aligning duplicate report suppression in *count mode* with that of *loop mode*?

gsnw-sebast reviewed (approved):
From my point of view, we can take it from there.

auerswal commented:
While rebasing, I noticed that I had introduced blank lines between changelog items in a block. This was inconsistent with older changelog entries, so I removed those blank lines while solving the `CHANGELOG.md` merge conflict. I hope that is OK.
